New RSTUF Guide (#341)
repository-service-tuf/repository-service-tuf: docs/source/guide/deployment/planning/volumes.rst

#######
Volumes
#######

As the Repository Service for TUF provides containers images, it is required to

> work with volumes for persistent data.

Reviewer: Is this multiple containers?  I believe in this instance we can still use a single term of "container".

Author: We have two container images: rstuf-worker and rstuf-api

.. Note::
    For the :ref:`guide/deployment/planning/deployment:Required Infrastructure Services`,
    use persistent data.

Repository Service for TUF API (RSTUF API)
##########################################

DATA [optional]
===============

It requires a persistent volume only if using the built-in
authentication/authorization.

.. Caution::

>     We don't recommend using the built-in RSTUF
>     :ref:`guide/deployment/planning/deployment:Authentication/Authorization`.

Reviewer: ```suggestion
    Using the built-in RSTUF
    :ref:`guide/deployment/planning/deployment:Authentication/Authorization` is not recommended.
```
